### PR TITLE
Implemented Stack Using Queue in C

### PR DESCRIPTION
**Pull request by sumitgsh**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
YES
### Description

(Write your answer here.)
Implemented Stack Using Queue in C
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
Issue #2244

### CODE REVIEW

**Comment by sumitgsh** [comment]
@siddharth25pandey @HarshCasper @iamrajiv please review this.

File: C/ds/StackUsingQueue.c [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+int main() {
+
+  int i;
+  push(1);
+  push(2);
+  push(3);
+  push(4);
+  push(5);
```
Comment (siddharth25pandey) [inline C/ds/StackUsingQueue.c]: Insert dynamic Input from the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@sumitgsh Do the changes as suggested

File: C/ds/StackUsingQueue.c (hunk 1 above)
Comment (sumitgsh) [reply C/ds/StackUsingQueue.c]: @siddharth25pandey can u review my changes

**Review by siddharth25pandey — APPROVED** [review]
@sumitgsh LGTM, able to merge 👍

**Review by iamrajiv — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by sumitgsh** [comment]
@HarshCasper @ankitaggarwal23  can u please add GSsco21 label and level as well to the merged PR

**Comment by HarshCasper** [comment]
@all-contributors bot please add @sumitgsh for code

**Pull request merged**